lib/features/steps/error_reporting_steps.rb

```diff
@@ -61,6 +61,16 @@
   end
   fail("No breadcrumb matched: #{value}") unless found
 end
+Then("the event does not have a {string} breadcrumb named {string}") do |type, name|
+  value = Server.current_request[:body]["events"].first["breadcrumbs"]
+  found = false
+  value.each do |crumb|
```

Comment:
We can use none? or any? (depending on what you want the check at the end to be) https://ruby-doc.org/core-2.6.2/Enumerable.html#method-i-none-3F

Reply:
I was thinking this could be simplified anyway, as the 4 events all require very similar logic.  I'll refactor before this is merged.